demos/tutorials/6_fluid_tank.ipynb

```diff
@@ -0,0 +1,296 @@
+        "\n",
+        "The fluid tank is a cylinder of radius $R_\\mathrm{tank}$ and height $h_\\mathrm{tank}$, initially filled with a height $h_\\mathrm{fluid}$ of fluid. New fluid is injected with velocity $v_\\mathrm{inlet}$ into the tank from a circular inlet at the top of the container with arbitrary coordinates $(x_\\mathrm{inlet}, y_\\mathrm{inlet})$. The fluid exits the tank from a cylindrical outlet at the bottom of the container.\n",
```

Comment:
A tank and a container are the same thing right?

Reply:
Yep. We use these terms interchangeably (mainly to avoid repeating the same word multiple times in a sentence).